BULK VALIDATION description-missing (#1430)
dotnet/docs-desktop: dotnet-desktop-guide/framework/winforms/controls/checkbox-control-windows-forms.md

---
title: "CheckBox Control"

> description: Learn about the CheckBox control in Windows Forms, which indicates whether a particular condition is on or off.

Reviewer: ```suggestion
description: Learn about what articles can help you understand more about the CheckBox control in Windows Forms.
```
